doc/source/releases/21.01_announce.rst

```diff
@@ -1,11 +1,82 @@
+
+**Workflow Enhancements and Fixes**
+  This release contains many new features, enhancements and bug fixes for creating, running and reporting workflows described in greater details in the `21.01 user release notes <21.01_announce_user.html>`__
+
+**History Import and Export**
+  Histories can now be imported from and exported to the pluggable file source architecture introduced in the 20.09 release of Galaxy.
+  This includes the users' FTP folder, S3, dropbox, owncloud etc. In addition the History Export and Import dialogues have been rewritten
+  to provide access to these new options and to provide more feedback during Import and Export process.
+
+Also check out the `21.01 user release notes <21.01_announce_user.html>`__
+
+Deprecation Notices
+===========================================================
+
+**Deprecation of support for Python 3.5**
+  Galaxy requires Python 3.6 or newer. Instruction for updating Python can be found in the `Supported Python versions <https://docs.galaxyproject.org/en/release_21.01/admin/python.html#supported-python-versions>`__ section of the documentation.
+  Note that if you update to Python 3.7.2 or newer you need to set py-call-osafterfork to ``false`` in the uwsgi section of galaxy.yml.
+
+**Removal of IUC conda channel from default configuration**
+  The IUC conda channel has been removed from the default configuration. This channel was used for installing dependencies that were old and not available in bioconda or conda-forge. These dependencies are now available in Bioconda. By removing this channel Conda can work more efficiently and ignore outdated dependencies.
+  If you have specified alternative channels in your Galaxy configuration we recommend remving the iuc channel from the `conda_ensure_channels <https://github.com/galaxyproject/galaxy/blob/283ec1772fcc0e574997dc2c5ad12dc86e3e1d37/lib/galaxy/config/sample/galaxy.yml.sample#L324>`__ variable.
+
+**Removal of the communication server**
+  This feature had not seen greater adoption and was now removed. Most communication has moved to Gitter or other chat services.
+
+Get Galaxy
+==========
+
+The code lives at `GitHub <https://github.com/galaxyproject/galaxy>`__ and you should have `Git <https://git-scm.com/>`__ to obtain it.
+
+To get a new Galaxy repository run:
+  .. code-block:: shell
+
+      $ git clone -b release_21.01 https://github.com/galaxyproject/galaxy.git
+
+To update an existing Galaxy repository run:
+  .. code-block:: shell
+
+      $ git fetch origin && git checkout release_21.01 && git pull --ff-only origin release_21.01
+
+See the `community hub <https://galaxyproject.org/develop/source-code/>`__ for additional details regarding the source code locations.
```

Comment:
```suggestion
See the `community hub <https://galaxyproject.org/develop/source-code/>`__ for additional details on source code locations.
```

Reply:
Can you also update the template in scripts/bootstrap_history.py ?

Reply:
Done.